app/classifier/tasks/dropdown/index.spec.js

```diff
@@ -81,6 +88,16 @@ describe('DropdownTask:static methods', function () {
 });
 
 describe('DropdownTask', function () {
+  describe('with single select', function () {
+    const annotation = { value: [{ value: 0, option: true }] };
+
+    const wrapper = mount(<DropdownTask task={singleSelect} translation={singleSelect} annotation={annotation} onChange={function (a) { return a; }} />, mockReduxStore);
+
+    it('should show that 0 is selected', function () {
+      assert.equal(wrapper.find('#numbers123').find('[role="option"][aria-selected="true"]').text(), '0');
```

Comment:
It might be safer here to expect that the value of the `Select` component matches the selected option. Then your test would be independent of how the `Select` component is rendered.

Reply:
Thanks @eatyourgreens great suggestion, I think updated accordingly.